### DISCUSSION TITLE
How can I request a change to devs in order to make a clickable elment "hint-able" in qutebrowser?

### DISCUSSION THREAD

**Discussion by Aster89** [qutebrowser/qutebrowser#7236] [Q&A] [answered]
There's an internal app I use in my company where threre's a clickable element that copies stuff to clipboard.

Based on my quick inspection, the elment is a `div` with a `class="do-something-on-click"`, some CSS code to show the pointer cursor, and some JavaScript to do the trick.

So it looks and behaves like a link via mouse interaction, but there's no hint in qutebrowser about it. I know this is normal, but I was wondering what should I tell the devs of that internal app to get this thing for me working?

**Comment by The-Compiler** [comment, marked as answer]
Pretty much anything that would also make it accessible (either via keyboard, or for blind users) on regular browsers. For example, adding a [tabindex](https://web.dev/using-tabindex/) to make it keyboard-focusable via tab in normal browsers. Or adding a [button ARIA role](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/button_role) (`role="button"`) to it.

**Reply by The-Compiler** [reply]
Ah, note that you can also modify `hints.selectors` for that page (via `config.py`) so that `div` elements with `class="do-something-on-click"` get hints.

**Reply by Aster89** [reply]
The ARIA role solution seems the most appropriate, thanks! :smile:
